lib/Doctrine/DBAL/Schema/OracleSchemaManager.php

```diff
@@ -382,4 +382,18 @@ private function killUserSessions($user)
             );
         }
     }
+
+    public function listTableDetails($tableName)
```

Comment:
This method should probably have at least a return value, correct?

Reply:
Absolutely :+1:

Reply:
Please do the same for all methods below then :P

Reply:
Ooops, my bad :)
Done in https://github.com/doctrine/dbal/pull/3512/commits/fc053b1e1a6ceeba6954eedef5269f5cbb93d03d